signac/core/synced_collections/collection_json.py

```diff
@@ -111,6 +112,8 @@ def _load_from_resource(self):
         except OSError as error:
             if error.errno == errno.ENOENT:
                 return None
+            else:
+                raise error
```

Comment:
See above discussion of re-raising. Not 100% sure if we want to make this change or not.
```suggestion
                raise
```

Reply:
Yes, let's just `raise`.